### PR TITLE
fix(localizations): Review PT-BR localization

### PR DESCRIPTION
**Pull request by danilofuchs**
## Description

Adds missing keys, fixes some typos

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Comment by LauraBeatris** [comment]
@danilofuchs Obrigada pela contribuição! Could you add a changeset describing your changes? This can be achieved by running `npm run changeset`

**Comment by danilofuchs** [comment]
> @danilofuchs Obrigada pela contribuição! Could you add a changeset describing your changes? This can be achieved by running `npm run changeset` - refer to [CONTRIBUTING.md](https://github.com/clerk/javascript/blob/29bf434d809eb062165932b06741b55d456d47bf/docs/CONTRIBUTING.md#changesets)

Imagina!
Sure, I added it

File: .changeset/great-parents-invent.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/localizations': patch
+---
+
+Review PT-BR localization
```
Comment (LauraBeatris) [inline .changeset/great-parents-invent.md:5]: Can we both update the PR title + changeset to mention "Fix typos and add missing keys to pt-BR localization"?

"Review" is a bit generic for the changes introduced here.

File: packages/localizations/src/pt-BR.ts [hunk 2]
```diff
@@ -354,7 +358,7 @@ export const ptBR: LocalizationResource = {
     noAvailableMethods: {
       message: 'Não foi possível fazer login. Não há nenhum método de autenticação disponível.',
       subtitle: 'Aconteceu um erro',
-      title: 'Não foi possível fazer login',
+      title: 'Não foi possível entrar',
```
Comment (LauraBeatris) [inline packages/localizations/src/pt-BR.ts:361]: Just to clarify the change here - why is the reason of replacing "fazer login" with "entrar"?

Comment (danilofuchs) [reply packages/localizations/src/pt-BR.ts:361]: I thought I reverted this change, sorry!
Realized fazer login is the nomenclature everywhere

**Review by LauraBeatris — APPROVED** [review]

**Review by nikosdouvlis — APPROVED** [review]

**Comment by nikosdouvlis** [comment]
Many thanks for your contribution @danilofuchs :)

**Pull request merged**